test/bin/build_images.sh

```diff
@@ -571,6 +573,17 @@ is_rhocp_available() {
     return 1
 }
 
+is_rhocp_beta_available() {
+    local -r ver="${1}"
+    url_arm="https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rpms/4.${ver}-el9-beta/"
+    url_amd="https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rpms/4.${ver}-el9-beta/"
+    if sudo dnf -v repository-packages --disablerepo '*' --repofrompath "this,${url_arm}" this info cri-o 1>&2 && \
+        sudo dnf -v repository-packages --disablerepo '*' --repofrompath "this,${url_amd}" this info cri-o 1>&2; then
```

Comment:
Can we avoid using -v here? It generates a lot of messages that may not be very informative in general.